src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -424,7 +424,11 @@ internal Accessibility LocalAccessibility
         /// </summary>
         internal override bool IsDeclaredReadOnly => LocalDeclaredReadOnly || _property.HasReadOnlyModifier || IsReadOnlyAutoGetter;
```

Comment:
The properties `IsReadOnlyAutoGetter` and `IsBadNetModule` are both only used in the implementation of this property. At this point I  would expand this out and make them local functions. Avoids member pollution.

Reply:
Done